pallet/account-migration/src/lib.rs

```diff
@@ -203,7 +195,7 @@ pub mod pallet {
 				// https://github.dev/paritytech/substrate/blob/19162e43be45817b44c7d48e50d03f074f60fbf4/frame/vesting/src/lib.rs#L86
 				<pallet_balances::Pallet<T>>::set_lock(*b"vesting ", &to, locked, reasons);
 			}
-			if let Some(l) = <Bonded<T>>::get(&from).and_then(<Ledgers<T>>::get) {
+			if let Some(l) = <Ledgers<T>>::get(&from) {
```

Comment:
It should be `take(from)`  instead of `get(from)`.

- `<Vestings<T>>::get(&from)`
- `<Ledgers<T>>::get(&from)`
- `<Deposits<T>>::get(&from)`